karate-core/src/main/java/com/intuit/karate/driver/Driver.java

```diff
@@ -445,6 +445,15 @@ default List scriptAll(String locator, String expression, Predicate predicate) {
     @AutoDef
     public byte[] pdf(Map<String, Object> options);
 
+    @AutoDef
+    default void setCookies(Map<String,Map> cookiesMap) { // method to allow set cookies from feature file for UI tests.
```

Comment:
great, I would actually try to use the current URL e.g. `driver.getUrl()` to derive the "correct" URL, but ok for now

actually I am itching to create a much better way to test the UI instead of the "monolith" `driver/core/test-01.feature` let me do some thinking over that in the next few days. meanwhile I can check in this commit as is and make any changes over it no worries

Reply:
@ptrthomas - sure thing, will work on it today. Thank you.